app/src/main/java/com/github/sdp_begreen/begreen/Photo.kt

```diff
@@ -0,0 +1,50 @@
+package com.github.sdp_begreen.begreen
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.os.Build
+import android.os.Parcel
+import android.os.Parcelable
+import androidx.core.content.ContextCompat
+import androidx.core.graphics.drawable.DrawableCompat
+import androidx.test.platform.app.InstrumentationRegistry
+
+
+data class Photo(val key: String?, val takenOn: ParcelableDate?, val takenBy: User?, val category: String?) : Parcelable {
+    constructor(parcel: Parcel) : this(
+        parcel.readString(),
+        parcel.readParcelable(ParcelableDate::class.java.classLoader),
+        parcel.readParcelable(User::class.java.classLoader),
+        parcel.readString()
+    ) {
+    }
+
+    fun getPhotoFromDataBase() : Bitmap? {
+        //TODO : get the photo from the database and maybe cache?
+        return Bitmap.createBitmap(100, 100, Bitmap.Config.ARGB_8888)
```

Comment:
Quite surprised that "Bitmap.Config.ARGB_8888" worked for you. I had some troubles with it.

Reply:
Oh ok we could see together IRL why it doesn't work with you although I don't really know if it works as it is just a Bitmap placeholder for now as it needs to be implemented =D